src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -491,10 +496,10 @@ class Alpha_complex {
         std::clog << vertex << " ";
       }
       std::clog << "is a face of Sigma\n";
-      std::clog << " | isnan(complex.filtration(Tau)=" << std::isnan(complex.filtration(f_boundary)) << std::endl;
+      std::clog << " | isnan(complex.filtration(Tau)=" << isnan(complex.filtration(f_boundary)) << std::endl;
```

Comment:
Wasn't your `using std::isnan;` in another function?

Reply:
> Is it ok to put it at `Gudhi::alpha_complex` namespace ?

Yes, I don't think it is likely to cause much trouble there.

> Or I repeat it in the 2 methods that are using it, as it cannot be done at class level.

Or that, as you prefer.

Reply:
Ok, let's go with repeating it in the 2 methods ([f772fb1](https://github.com/GUDHI/gudhi-devel/pull/798/commits/f772fb15dfe6f4fdaa99f2470b6f960a81aed84b)), it avoids embedding it when doing `using namespace Gudhi::alpha_complex`